### DISCUSSION TITLE
Welcome to Hacktoberfest2022 Discussions!

### DISCUSSION THREAD

**Discussion by ossamamehmood** [ossamamehmood/Hacktoberfest#54] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jc8223** [comment]
@ossamamehmood i have query that i have got hacktoberfest label in my pr and it is got merged but on my profile it is showing EXCLUDED not counted why??

**Reply by azizarizaldi** [reply]
yups, me too.

**Comment by ossamamehmood** [comment]
We have created a new repository : 
https://github.com/ossamamehmood/Hacktoberfest2023

Note: This current repo was excluded due to high volume spam and low-quality contribution. So, we set up a new repo that follows #hacktoberfest guidelines.

Additionally, we acknowledge fair and quality contributions. You must follow Hacktooberfest contribution guidelines to approve your Pull Request!

Special thanks to our maintainer's team who are always trying to improve the quality of Hacktoberfest events from now onwards.
